### PR TITLE
aws-history: add page

### PR DESCRIPTION
**Pull request by piraces**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 2.7.25

Related: #5653

### CODE REVIEW

File: pages/common/aws-history.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# aws history
+
+> Commands to interact with the history of AWS CLI commands ran over time (the record of history of AWS CLI commands must be enabled).
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/history/>.
+
+- Get a list of previously run commands and their command ids:
```
Comment (navarroaxel) [inline pages/common/aws-history.md:6]: Perhaps this better
```suggestion
- List commands history with command IDs:
```

File: pages/common/aws-history.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`aws history list`
+
+- Get the various events related to running a specific CLI command with its command id:
+
+`aws history show {{1affe293-ed81-451c-a044-ea99223a93bf}}`
```
Comment (navarroaxel) [inline pages/common/aws-history.md:12]: Perhaps a placeholder is better in this example
```suggestion
`aws history show {{command_id}}`
```

File: pages/common/aws-history.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# aws history
+
+> Commands to interact with the history of AWS CLI commands ran over time (the record of history of AWS CLI commands must be enabled).
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/history/>.
+
+- Get a list of previously run commands and their command ids:
+
+`aws history list`
+
+- Get the various events related to running a specific CLI command with its command id:
```
Comment (navarroaxel) [inline pages/common/aws-history.md:10]:
```suggestion
- Display events related to a specific command given a command ID:
```

File: pages/common/aws-history.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# aws history
+
+> Commands to interact with the history of AWS CLI commands ran over time (the record of history of AWS CLI commands must be enabled).
```
Comment (navarroaxel) [inline pages/common/aws-history.md:3]:
```suggestion
> Print the command-line history for AWS CLI commands (the record of history of AWS CLI commands must be enabled).
```

Comment (piraces) [reply pages/common/aws-history.md:3]: Thanks for all suggestions! 😄

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**